CanvasColorSpaceProposal.md

```diff
@@ -95,13 +96,16 @@ partial dictionary CanvasRenderingContext2DSettings {
 };
 </pre>
 
-* All input colors (e.g, ``fillStyle`` and ``strokeStyle``) follow the same interpretation as CSS color literals, regardless of the canvas color space.
-* Images drawn to a canvas are converted from their native color space to the backing storage color space of the canvas.
+2D Canvases are color managed.
+* All inputs that are drawn to a canvas have specific color space
+  * Input colors (e.g, ``fillStyle`` and ``strokeStyle``) follow the same interpretation as CSS color literals, regardless of the canvas color space.
   * Images with no color profile, when drawn to the canvas, are assumed to be in the sRGB color space.
-* All blending operations are performed in the canvas' backing storage color space.
+* All inputs that are drawn to a canvas are converted to the canvas' color space before being drawn to the canvas.
```

Comment:
Does this make the first bullet point obsolete?

Reply:
I changed it from "are converted to the canvas' color space" to "are converted from their color space to the canvas' color space". I wanted to make the points separately that there are no color-space-less inputs to a canvas, and that the conversion is done before drawing. I added a parenthetical about no-color-space-less inputs.